ocs_ci/templates/app-pods/eck1.7/crds.yaml

```diff
@@ -0,0 +1,4466 @@
+# Source: eck-operator-crds/templates/all-crds.yaml
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
can we comment where this file is coming from either in yaml or in python file?

Reply:
As discussed lets move this file outside ocs-ci,:https://github.com/red-hat-storage/ocs-workloads